### PR TITLE
bump(*) to v1.19.0-rc.3

### PR DESCRIPTION
**Pull request by gabemontero**
/assign @adambkaplan 

fyi @coreydaley @otaviof

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -8,26 +8,24 @@ require (
...
 	github.com/spf13/cobra v0.0.5
 	github.com/tmc/grpc-websocket-proxy v0.0.0-20190109142713-0ad062ec5ee5 // indirect
 	github.com/ugorji/go v1.1.7 // indirect
 	go.uber.org/zap v1.12.0 // indirect
-	google.golang.org/appengine v1.6.2 // indirect
-	k8s.io/api v0.18.0
-	k8s.io/apimachinery v0.18.0
-	k8s.io/client-go v0.18.0
+	k8s.io/api v0.19.0-rc.3
+	k8s.io/apimachinery v0.19.0-rc.3
```
Comment (gabemontero) [inline go.mod:27]: note, because of changes in openshift/library-go,

File: go.mod [hunk 2]
```diff
@@ -8,26 +8,24 @@ require (
...
+	k8s.io/apiserver v0.19.0-rc.3 // indirect
```
Comment (gabemontero) [inline go.mod:28]: because of changes in openshift/library-go, we have to explicitly note k8s.io/apiserver in order for `go mod vendor` to work.

**Comment by gabemontero** [comment]
e2e failures unrelated to samples or samples operator

/retest

**Comment by gabemontero** [comment]
ok all green tests @adambkaplan @coreydaley for the sample operator 1.19 rebase

File: vendor/modules.txt [hunk 3]
```diff
@@ -197,24 +207,25 @@ golang.org/x/net/idna
...
 golang.org/x/oauth2/jwt
 # golang.org/x/sync v0.0.0-20190911185100-cd5d95a43a6e
 golang.org/x/sync/singleflight
-# golang.org/x/sys v0.0.0-20191022100944-742c48ecaeb7
+# golang.org/x/sys v0.0.0-20200622214017-ed371f2e16b4
+golang.org/x/sys/internal/unsafeheader
 golang.org/x/sys/unix
 golang.org/x/sys/windows
-# golang.org/x/text v0.3.2
+# golang.org/x/text v0.3.3
```
Comment (adambkaplan) [inline vendor/modules.txt:222]: Note that this bumps golang.org/x/text, thereby fixing CVE-2020-14040 (not sure if we have a BZ for the samples operator on this one).

Comment (gabemontero) [reply vendor/modules.txt:222]: thanks @adambkaplan 

we do have a BZ and I've associated this PR with it, though through the BZ UI vs. marking this PR ... was planning on moving that BZ to MODIFIED manually if need be once this PR merged

could pivot on that and use that BZ here with this PR if we want ... I'm fine either way

**Pull request merged**